IlmBase/ImathTest/testFun.cpp

```diff
@@ -51,9 +51,24 @@ using namespace std;
     typedef long long unsigned int Int64;
 #endif
 
+template <typename To, typename From>
+static inline To
+bitwise_cast (From from)
+{
+#if __cplusplus >= 201103L
+    static_assert (sizeof (From) == sizeof (To), "Type sizes do not match");
+#endif
```

Comment:
C++11 is our minimum version, so this `#if` guard is not needed.

Reply:
`#if` has now been removed.